### DISCUSSION TITLE
为什么dolphinscheduler不支持子工作流实例的查询呢  版本3.1.8

### DISCUSSION THREAD

**Discussion by shashale2** [apache/dolphinscheduler#15550] [Q&A]
我建立了一个父工作流，使用 sub_process节点来支持子工作流的依赖。  当我运行这个父工作流之后，在工作流实例里只能看到父工作流的实例， 无法看到子工作流的实例。 为此 我查看dolphin scheduler的代码，发现查看工作量实例接口特意屏蔽了子工作流实例。这是处于什么考虑呢 

<img width="1057" alt="image" src="https://github.com/apache/dolphinscheduler/assets/34544770/7ebbde0c-49eb-49e4-b8e6-fb2bba6eb3eb">

**Comment by shashale2** [comment]
<img width="1012" alt="image" src="https://github.com/apache/dolphinscheduler/assets/34544770/445640e1-70a0-431f-b168-3a6ca61994a6">

**Comment by CHINAJAVABoy** [comment]
因为代码很难写，子流程中又可能有子流程，你的工作甘特图亲妈炸裂
